dotnet/src/dotnetcore/Providers/OpenTelemetry/OpenTelemetry/GXMeterProviderBuilder.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace GeneXus.OpenTelemetry
+{
+	public static class GXMeterProviderBuilder
+	{
+		public static MeterProviderBuilder AddGxMeterAspNetInstrumentation(this MeterProviderBuilder meter)
+		{
+			meter
+				.AddAspNetCoreInstrumentation()
+				.AddHttpClientInstrumentation()
+				.AddRuntimeInstrumentation()
+				.AddOtlpExporter();
+			if (Environment.GetEnvironmentVariable("OTEL_METRICS_EXPORTER")?.ToLower() == "console")
```

Comment:
We should use "includes" instead of ==. 

Because a valid value would be: 
OTEL_METRICS_EXPORTER=console,oltp,zipkin.

//The implementation MAY accept a comma-separated list to enable setting multiple exporters.